bible

```diff
@@ -5,14 +5,15 @@ from bs4 import BeautifulSoup
 import argparse
 import requests
 from unidecode import unidecode
+import urllib.parse
 
 
 def parse_args():
     default_version = "NIV"
     parser = argparse.ArgumentParser(
         description="Fetch Bible references from biblegateway.com"
     )
-    parser.add_argument("reference", nargs="+", help="bible reference")
+    parser.add_argument("reference", nargs="*", help="bible reference")
```

Comment:
How does this change the behavior?

Reply:
actually we might not need this, but I need to re-check